source/includes/_240-secret-configs.md.erb

```diff
@@ -0,0 +1,5 @@
+# Secret Configs
+
+The secret configs API allows users with admin authorization to manage secret configurations.
```

Comment:
How bout?

The secret configs API allows users with admin privileges to configure and manage external Secret Managers.